features/template-literals.yml

```
@@ -0,0 +1,7 @@
+name: Template literals
+description: Template literals are literals delimited with backtick (`) characters, allowing for multi-line strings, string interpolation with embedded expressions, and special constructs called tagged templates.
```

For clarity, to show that we mean the character alone and that this is not some markdown issue.

```suggestion
description: Template literals are literals delimited with backtick (<code>`</code>) characters, allowing for multi-line strings, string interpolation with embedded expressions, and special constructs called tagged templates.
```